app/views/evaluation_forms/_form.html.erb

```diff
@@ -41,16 +41,33 @@
     <li class="usa-process-list__item">
       <h4 class="usa-process-list__heading">Evaluation Criteria</h4>
       <p>
-        Create your evaluation criteria and scoring options.
+        Build your evaluation criteria and scoring options.
       </p>
-      <div>
-        <%= form.label :comments_required, style: "display: block" %>
-        <%= form.check_box :comments_required %>
+      <div class="usa-checkbox">
+        <%= form.check_box :comments_required, class: "usa-checkbox__input", id: "evaluation_form_comments_required" %>
+        <label class="usa-checkbox__label" for="evaluation_form_comments_required">
+          Require evaluators to provide comments on their scores. (optional)
+          <%= image_tag 'help_outline.svg', alt: 'Help for comments required' %>
+        </label>
       </div>
 
       <div>
-        <%= form.label :weighted_scoring, style: "display: block" %>
-        <%= form.check_box :weighted_scoring %>
+        <fieldset class="usa-fieldset">
+          <legend class="usa-legend usa-process-list__heading eval-scale-heading">
+            Scale Type <span style="color: red;">*</span>
+            <%= image_tag 'help_outline.svg', alt: 'Help for scale type' %>
+          </legend>
+          <div class="radio-group">
```

Comment:
I'm not sure if radio-group is needed anymore. Unfortunately USWDS doesn't have the `gap` rule but `margin-right-205` on the .usa-radio element seems to accomplish the same
```suggestion
          <div class="display-flex flex-row">
```